source/index.md

```diff
@@ -45,3 +45,59 @@ eleventyComputed:
     <img height="160" alt="Glasses" src="/assets/img/illustrations/glasses.svg">
   </p>
 </div>
+
+<!-- @@@ TODO: remove these sample code examples -->
```

Comment:
Is this to be handled in a different card?

Reply:
@mirisuzanne I have it listed as one of the checklist items for this story. I don't think the linter is catching all of these `@@@`/`TODO` comments as expected.

Reply:
I think that's right -- ESLint only checks JS/TS files, and that's what highlights the TODO comments for us.

I think we can just remove this as the last step before merging the PR.